build/build_test.go

```diff
@@ -2,12 +2,12 @@ package build
 
 import (
 	"fmt"
-	gobuild "go/build"
 	"go/token"
 	"strconv"
 	"strings"
 	"testing"
 
+	gobuild "github.com/gopherjs/gopherjs/go/build"
 	"github.com/kisielk/gotool"
 	"github.com/shurcooL/go/importgraphutil"
```

Comment:
This doesn't fix the issue completely because `github.com/shurcooL/go/importgraphutil` package still uses original `go/build`, so it's incompatible.